server/settings/tools.py

```diff
@@ -222,6 +222,13 @@ def _product_types_enum():
     ]
 
 
+def filter_type_enum():
+    return [
+        {"value": "is_allow_list", "label": "Allow list"},
+        {"value": "is_deny_list", "label": "Deny list"},
```

Comment:
Could this have `value` True and False and have the type be `bool` whilst still being an Enum (for niceness)?

Reply:
Will try out...

Reply:
Nope, it will become checkbox.

Reply:
Ugh. :( I just really really dislike the usage of strings for this. It feels so darn nasty just so that we can stick a label on it for the user in the settings frontend.

I wonder if we shouldn't just keep it a checkbox for now so that the data is as we want it to be, yet open a frontend/backend issue to track and allow better options for checkboxes in that regard? (Like e.g. enum resolver with two options)

Thoughts?

Reply:
Fine for me, I just wanted to change Exclude/Include to something that is understandable un human language 😄

Reply:
Hmm keep it as is I suppose. It looks fine in setings, we'll figure out better settings design in the future maybe...